types/fake-response/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for fake-response 3.2
+// Project: https://github.com/R35007/Fake-Response
+// Definitions by: r35007 <https://github.com/R35007>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export as namespace FakeResponse;
+
```

Comment:
```suggestion
export as namespace FakeResponse;

```
Is your intended export an UMD module to be used in browser with global context?
If not remove that line:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#optional-global-usage